opal/mca/accelerator/rocm/accelerator_rocm_module.c

```diff
@@ -483,3 +486,9 @@ static int mca_accelerator_rocm_device_can_access_peer(int *access, int dev1, in
 
     return OPAL_SUCCESS;
 }
+
+static int accelerator_null_get_buffer_id(int dev_id, const void *addr, opal_accelerator_buffer_id_t *buf_id)
```

Comment:
this should probably be accelerator_rocm_get_buffer_id (not null_get_buffer_id), otherwise things look good.

Reply:
ah yes, will change